### PR TITLE
This commit adds a stop functionality to the existing REST API in the…

### PR DESCRIPTION
**Pull request by blangus**
… BurpController.java file. A GET request to /burp/stop will quit Burp Suite.

### CODE REVIEW

**Comment by gangelino** [comment]
@blangus we're very eager to merge your pull request, please sign the contributor agreement when you get a chance, thanks!

**Comment by blangus** [comment]
Sorry for the delay guys, I was sick a couple days. I will sign it right now!

**Comment by dmettem** [comment]
@blangus The reason why the stop functionality is not added in first place is that, once the /burp/stop is called, the Burp suite is exited, and none of the REST endpoints are accessible any more. While this is a nice to have endpoint to shutdown the Burp Suite, one should understand that there is no going back once the /burp/stop is called.

File: src/main/java/com/vmware/burp/extension/web/BurpController.java [hunk 1]
```diff
@@ -328,6 +328,13 @@ public void sendToSpider(@RequestParam String baseUrl)
       burp.sendToSpider(baseUrl);
    }
 
+   @ApiOperation(value = "Stop Burp Suite", notes = "This will exit Burp Suite.")
+   @RequestMapping(method = GET, value = "/stop")
+   public void exitBurp(){
```
Comment (dmettem) [inline src/main/java/com/vmware/burp/extension/web/BurpController.java]: Please see my comment in the pull request and if you still think it is definitely useful to have stop functionality, please add all the possible ApiResponse types for the swagger documentation purpose.

Thanks!

**Comment by blangus** [comment]
I use this endpoint in Jenkins. I have a set-up where I start Burp Suite from command line shell through Jenkins, start with Selenium tests, scan with Burp and afterwards close Burp by calling the /stop endpoint. 

I have added the api responses. If you'd like you can merge the PR.

**Pull request merged**